app/views/casework/crime_applications/index.html.erb

```diff
@@ -1,59 +1,26 @@
 <% title t(".#{action_name}_title") %>
 
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-full">
-    <h1 class="govuk-heading-xl govuk-!-margin-bottom-3">
-      <%= t ".#{action_name}_heading" %>
-    </h1>
-  </div>
-</div>
+<% if FeatureFlags.work_stream.enabled? %>
+  <div class="govuk-tabs">
+    <ul class="govuk-tabs__list" role="tablist">
+      <% CaseworkerPresenter::CASEWORKER_COMPETENCIES.each_with_index do |competency| %>
+        <% css_class = %w[govuk-tabs__list-item govuk-tabs__list-item--selected] %>
```

Comment:
I wonder if this should be moved into a tab view component.

Reply:
Good point